release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1559,6 +1559,36 @@ For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=19300
 
 * Previously, an incorrect code was causing `User` and `Group` details to show unrelated subjects. Now, a code has been added to filter by `User` or `Group`, so the `User` and `Group` details show related subjects. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1951212[*BZ#1951212*])
 
+* Previously, when you imported a devfile, the Openshift web console ignored the `build guidance` placeholder container because the new deployment had a second container. The web console could not start the container because it could not fetch the placeholder image. As a result, the user container missed the additional configurations. Now, in the web console, the `build guidance` container is dropped from the new deployment, and the container adds the environment variable, ports, and limit configurations. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1952214[BZ#1952214])
+
+* Previously, the *Topology* view showed an error or a blank page. This happened because the OpenShift Samples Operator loads page templates to the OpenShift namespace, and was not installed. The current release fixes this issue. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1949810[BZ#1949810])
+
+* Previously, routes tied to the *Developer* perspective were not rendered. This produced 404 error messages when you refresh the Developer perspective. The current release fixes this issue by loading all the inactive routes. Now, the routes switch to the correct perspective before loading. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1929769[BZ#1929769])
```

Comment:
```suggestion
* Previously, routes tied to the *Developer* perspective were not rendered. This produced 404 error messages when you refreshed the Developer perspective. The current release fixes this issue by loading all the inactive routes. Now, the routes switch to the correct perspective before loading. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1929769[BZ#1929769])
```

Reply:
Developer needs to be in bold here -  when you refreshed the Developer perspective.